site/docs/guides/spark-s3.md

```diff
@@ -54,7 +54,52 @@ for the most up-to-date information on how to connect Iceberg to S3.
 Note: the word `nessie` in configuration property names is the name of the Nessie catalog in the Spark session.
 A different name can be chosen according the user's liking.
 
-Then, in `spark-sql` issue a `use` statement to make `nessie` the current catalog:
+## Setting up Spark Session for Minio
+
+Start a local Minio server using Docker:
+
+```shell
+docker run -p 9000:9000 -p 9001:9001 --name minio \
+ -e "MINIO_ROOT_USER=datauser" -e "MINIO_ROOT_PASSWORD=minioSecret" \
+ quay.io/minio/minio:latest server /data --console-address ":9001"
+```
+
+Configure an AWS profile (e.g. called `demo`) in `~/.aws/credentials` (or other location appropriate for your OS) and 
```

Comment:
Wonder if the example should use `AWS_ACCESS_KEY_ID` + `AWS_SECRET_ACCESS_KEY` environment variables for simplicity over the profile approach (same for the AWS S§ walkthrough above).

Could add a link to https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html if people are interested how keys are looked up.

Reply:
+1 for presenting 2 different methods, and also +1 for linking users to official docs (the link @snazy gave is even better than mine).

Reply:
TBH, I'm hesitant to use env. vars containing passwords in guides like this as I think that practice is not ideal from the security POV. I much prefer having credentials in a local file, which I think offers stronger protection.